src/db/db.types.ts

```diff
@@ -110,6 +110,33 @@ type UserDataRow = {
   value: string
 }
 
+// SIGNERS -----------------------------------------------------------------------------------------
+declare const $signerDbId: unique symbol
+type SignerDbId = string & { [$signerDbId]: true }
+
+export type SignerAddMetadataJson = {
+  requestFid: number
+  requestSigner: Hex
+  signature: Hex
+  deadline: number
+}
+
+type SignerRow = {
```

Comment:
wdyt about making some of these column names more consistent with other tables? so `timestamp` instead of `addedAt` for the time of the event, `deletedAt` instead of `removedAt`, etc.

also is `addChainEventId` and `removeChainEventId` relevant? Noticed they are commented out